src/message.ts

```diff
@@ -414,6 +414,67 @@ export class Message implements Sayable {
     return fromId === userId
   }
 
+  /**
+   *
+   * Get the mentioned contact which the message is mentioned for.
+   * @returns {Contact[]} return the contactList which the message is mentioned for
+   *
+   * @example
+   * ```ts
+   * const content = message.content()
+   * const contactList = message.mentioned()
+   * console.log(content)
+   * console.log(contactList)
+   * ```
+   */
+  public mentioned(): Contact[] {
+    let contactList: Contact[] = []
+    const room = this.room()
+    if (this.type() !== MsgType.TEXT || !room ) {
+      return contactList
+    }
+
+    const atStrings = this.content().split(String.fromCharCode(8197))
+
+    if (atStrings.length === 0) return contactList
+
+    const strings = atStrings.filter(e => e.indexOf('@') > -1).map(e => {
+      const list = e.split('@')
+      if (list.length < 3) {
+        return [e.slice(e.lastIndexOf('@') + 1)]
+      } else {
+        // deal name: lijiarui@wechaty@beijing
+        let nickList: string[] = []
+        for (let i = 1; i < list.length; i++) {
+          let nick = ''
+          for (let j = 1; j <= i; j++) {
+            nick = '@' + list[list.length - j] + nick
+          }
+          nickList.push(nick.slice(1))
+        }
+        return nickList
+      }
+    })
```

Comment:
5. here to deal with multiple `@` string, the for cyclic to get the continuous  `@` string end with the `list` last element. for example, `e = 'Let me introduce @lijiarui@wechaty@beijing'`, then `list = ['Let me introduce ','@lijiarui','@wechaty','@beijing']`, after for cyclic, we get `nickList = ['beijing','wechaty@beijing','lijiarui@wechaty@beijing']`